### ISSUE TITLE
Eisenhower tag does not work

### ISSUE THREAD

**Issue by lucbeaulieu** [AlexanderWillner/KanbanView#24]
I cannot get any task in the Eisenhower matrix. I have tried using your current tags (A,...D) or the new option in the preference (I am using the official app!).

**Comment by lucbeaulieu** [comment]
> I cannot get any task in the Eisenhower matrix. I have tried using your current tags (A,...D) or the new option in the preference (I am using the official app!).

Also found out that when I wrote a custom get, getting out and in the preference seems to erase the custom tag I entered.

**Comment by AlexanderWillner** [comment]
Many thanks for the support and feedback. Seems it "works for me" in the released Version 2.6.1 (202005081019). Maybe following the guide at http://kanbanview.app/support helps to identify the root cause of the issue...

**Comment by lucbeaulieu** [comment]
I think it is because I use Emojis and Symbol as tags (for make things shorter). When I quit and reopen KanbanView it revert back to A, B, ....

**Comment by lucbeaulieu** [comment]
Well it only partially work.é According to my Things database, I should have tasks in each quadrant and over 12 in quadrant B. See the attach screen capture below.
[Capture d’écran, le 2020-05-18 à 16.44.08.pdf](https://github.com/AlexanderWillner/KanbanView/files/4646573/Capture.d.ecran.le.2020-05-18.a.16.44.08.pdf)

**Comment by AlexanderWillner** [comment]
Seems there is at least not an issue with symbols in general:

<img width="534" alt="Screen Shot 2020-05-18 at 23 17 25" src="https://user-images.githubusercontent.com/307605/82260557-cab3a300-995d-11ea-86d7-4faca3d21b6e.png">

Have you tested the latest version here in the repository? Pushed a potential fix.

**Comment by lucbeaulieu** [comment]
No I have been testing this option with the official App only (2.6.1). Will get the latest release from GIT and try tomorrow.





Luc

🌈 Ça va bien aller / It’s going to be OK

(Please excuse grammar. Sent from mobile.) 







Le 18 mai 2020 à 17:21, Alex <notifications@github.com> a écrit :


[Externe UL*]



Seems there is at least not an issue with symbols in general:

Have you tested the latest version here in the repository? Pushed a potential fix.
—
You are receiving this because you authored the thread.
Reply to this email directly, 
view it on GitHub, or 
unsubscribe.


*ATTENTION : L’émetteur de ce courriel est externe à l’Université Laval.

Évitez de cliquer sur un hyperlien, d’ouvrir une pièce jointe ou de transmettre des informations si vous ne connaissez pas l’expéditeur du courriel. En cas de doute, contactez l’équipe de soutien informatique de votre unité ou
hameconnage@ulaval.ca.

**Comment by lucbeaulieu** [comment]
Download 2.6.1, ran the api and same results. Not all of my tasks with Eisenhower tags appears in the Eisenhower matrix. DO you need particular conditions (most have multiple tags for example and I also regrouped my tags as a sub category call Eisenhower under which sits my four tags)?

**Comment by AlexanderWillner** [comment]
Mh, well, they've to be in Today to show up in the Eisenhower view. Maybe this is the issue?

**Comment by lucbeaulieu** [comment]
Ah, almost no utility then if you follow GTD. It would be much more useful to be able to use the Eisenhower matrix to plan what to put in the today view by including all the tasks (otherwise you can never be in the
Master and Commander part of the Eisenhower matrix, which is the whole point of the approach).

**Comment by AlexanderWillner** [comment]
Mh, might need to be an option. Temporarily changed the Eisenhower view to show all tasks. However, imho, if you've thousands of tasks, it probably doesn't make too much sense to use Eisenhower tags on many of them, as the urgency changes over time.

**Comment by lucbeaulieu** [comment]
Indeed. This is why it is a good idea to review tasks once a week and identify some key ones for the next 1-2 weeks to come. Having said that, the not important not urgent tasks should never end up in this matrix to start with since they would be eliminated during the weekly review process.

**Comment by lucbeaulieu** [comment]
> Temporarily changed the Eisenhower view to show all tasks. 

This is now very helpful in deciding what will be done any given day. During the meekly review, I make a tentative planning using these tags (quadrant D is of course never used, since it is either deferred to a later date or simply delete).

It would be very helpful visually if instead of display A, B, C and D, the title would be "Urgent and Important" and so on...
